### PR TITLE
Logitech G703 wired support

### PR DESCRIPTION
**Pull request by SonicFrog**
Adds a device file for the Logitech G703 connected through wired USB, as well as an SVG for the gnome theme.

Signed-off-by: Ogier Bouvier <obouvier@logitech.com>

### CODE REVIEW

File: data/gnome/logitech-g703.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+logitech-g403.svg
```
Comment (bentiss) [inline data/gnome/logitech-g703.svg]: Looks like this is not a SVG :)

Comment (Hjdskes) [reply data/gnome/logitech-g703.svg]: This is a symlink to logitech-g403.svg.

Comment (bentiss) [reply data/gnome/logitech-g703.svg]: Right, I have been mistaken by github's web ui.

In that case, wouldn't it be better to just have `logitech-g403.svg` in the `.device` file and add a comment saying both are the same. Having a separate file is just misleading IMO.

Comment (Hjdskes) [reply data/gnome/logitech-g703.svg]: There is at least one other symlink that follows the current approach, as far as I know.

**Pull request merged**